applications/API/serializers.py

```
@@ -0,0 +1,505 @@
+        
+class CategorySerializer(serializers.ModelSerializer):
+    category_key = serializers.ReadOnlyField(source="category_key")
+    name = serializers.ReadOnlyField(source="name")
+    insult_count = serializers.SerializerMethodField(method_name="get_count")
+    
+    class Meta:
+        model = InsultCategory
+        fields = "__all__"
+        
+    def get_count(self, instance):
+        return  Insult.objects.all(category_id=instance.category_key).count()
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)